documentation/modules/auxiliary/scanner/ip/ipidseq.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This ```auxiliary/scanner/ip/ipidseq``` module will probe hosts' IPID sequences and classify them using the same method Nmap uses when it's performing its IPID Idle Scan (-sI) and OS Detection (-O).
+
+More information: https://nmap.org/book/idlescan.html
+
+Nmap's probes are SYN/ACKs while this module's are SYNs.
+While this does not change the underlying functionality, it does change the chance of whether or not the probe will be stopped by a firewall.
+
+Nmap's Idle Scan can use hosts whose IPID sequences are classified as "Incremental" or "Broken little-endian incremental".
+
+## Verification Steps
+
+1. Start msfconsole
+1. Do: `use auxiliary/scanner/ip/ipidseq`
+1. Do: `set RHOSTS [ip]`
+1. Do: `run` or `exploit`
+
+## Options
```

Comment:
Keep heading, remove the code block lower than it.  I think in this case most things are self explanatory and can be left out. If you want, `SNAPLEN` could be described in more detail.